tools/webpack-make

```diff
@@ -27,7 +27,7 @@ var ops = stdio.getopt({
     watch: { key: "w", args: 0, description: "Enable webpack watch mode" },
 });
 
-var srcdir = process.env.SRCDIR || ".";
+var srcdir = process.env.SRCDIR || "./";
```

Comment:
would be nice to make the subject more descriptive: s/minor bugfix/fix default srcdir separator/ or so?

Reply:
The description of the problem in this commit message is actually incorrect.  I'll look into it again, but I'm also happy to just drop this commit.

Reply:
The problem is actually kinda funny: we iterate over input files and we check if filenames begin with the srcdir, and then remove the srcdir, plus one extra character (for the `/` that got inserted by path joining) from the start of the filename.

If srcdir is a bare `.` then this also matches `.babelrc` for example, and removes the `.` plus one extra character, landing us with `abelrc`.

I guess the real fix is actually to check if the string starts with `srcdir + '/'`.

Reply:
Okay.  Fixing this led me to fix a bunch of other things in `webpack-make`.  This PR is a big larger in scope now.